pkg/ocm/accounts_mgmt.go

```diff
@@ -103,6 +105,30 @@ func (a accountsMgmt) UpdateSubscription(ctx context.Context, subscriptionID, op
 	return responseVal, nil
 }
 
+func (a accountsMgmt) UpdateSubscriptionDisplayName(ctx context.Context, subscriptionID strfmt.UUID, displayName string) (*amgmtv1.Subscription, error) {
+	defer commonutils.MeasureOperation("OCM-UpdateSubscriptionDisplayName", a.client.log, a.client.metricsApi)()
+
+	// create the request
+	sub, err := amgmtv1.NewSubscription().DisplayName(displayName).Build()
+	if err != nil {
+		a.client.logger.Error(ctx, "Failed to create subscription request. Error: %v", err)
+		return nil, err
+	}
+
+	// send the request
+	response, err := a.client.connection.AccountsMgmt().V1().Subscriptions().Subscription(subscriptionID.String()).Update().Body(sub).SendContext(ctx)
+	err = HandleOCMResponse(ctx, a.client.logger, response, subscriptionPatchRequest, err)
+	if err != nil {
+		return nil, err
+	}
+	responseVal, ok := response.GetBody()
+	if !ok {
+		return nil, errors.Errorf("Empty response from %s request", subscriptionPatchRequest)
+	}
+
+	return responseVal, nil
```

Comment:
do we need to return the responseVal? seems we're not using it in the invocation.

Reply:
No, it was just for for consistency with other APIs, I will remove it.